games/v.yaml

```
@@ -454,6 +454,20 @@
   images:
   - https://static.wikia.nocookie.net/quake/images/4/4d/Vkquake0001.png/revision/latest/scale-to-width-down/1000?cb=20221204023831
 
+- name: voosweeper
+  originals:
+  - Minesweeper
+  type: clone
+  repo: 'https://github.com/M4rti21/voosweeper'
+  development: halted
+  status: playable
```

```suggestion
  status: semi-playable
```
it is not clear whether the game is fully playable based on the todo list